api/src/Controllers/AuthenticationController.php

```diff
@@ -300,7 +317,14 @@ private function returnResponse($code, $message)
 
     private function returnError($code, $message)
     {
-        $this->returnResponse($code, array('error' => $message));
+        $this->returnResponse(
+            $code,
+            array(
+                'error' => $message,
+                'user-agent' => $_SERVER['HTTP_USER_AGENT'],
+                'timestamp' => gmdate('c', time())
```

Comment:
I don't think we need to return these to the user. I think we should instead just log these when we log the auth token failure error and reason.